src/chrome/content/rules/QuranFlash.com.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Invalid Certificate:
+		v2.quranflash.com
```

Comment:
See also `v1` and `v3`.
Please add a hint about the wildcard DNS entry on this host.